episodes/disease-burden.Rmd

```diff
@@ -51,9 +60,20 @@ associated with the lessons. They appear in the "Instructor View".
 
 ## A burden model
 
-We will extend the influenza example in the tutorial [Simulating transmission](../episodes/simulating-transmission.md) to calculate hospitalisations over time. In this example our **transmission model** is an SEIR model comprised of ODEs. Our **burden model** will convert new infections to hospitalisations over time by summing up all the new hospitalisations we expect to occur on each day according to the delay from infection to hospitalisation. We can do this summation using a calculation known as a convolution.
+We will extend the influenza example from the [Simulating transmission](../episodes/simulating-transmission.md) tutorial to calculate hospitalizations over time. Our approach has two main components:
+
+1. **Transmission model**: An SEIR model that generates new infections over time
+2. **Burden model**: Converts new infections to hospitalizations by accounting for delays between infection and hospitalization
 
-We will first obtain the new infections through time from our influenza example (see tutorial on [Simulating transmission](../episodes/simulating-transmission.md) for code to generate `output_plot`) using `new_infections()` in `{epidemics}`
+To convert infections to hospitalizations, we need to consider:
+- The probability that an infection leads to hospitalization (infection-hospitalization ratio, IHR)
+- The time delay from infection to hospital admission
+- The time spent in hospital before discharge
+
+We'll use the `{epiparameter}` package to define these delay distributions. The Gamma distribution is commonly used for these delays because:
```

Comment:
to ease translation
```suggestion
We'll use `{epiparameter}` to define these delay distributions. The Gamma distribution is commonly used for these delays because:
```